scripts/bootstrap.sh.in

```diff
@@ -172,37 +172,62 @@ orchestrate_bootstrap() {
     # Grains must be set (in `/etc/salt/grains`) *before* invoking `salt-call`,
     # otherwise grains set during execution won't be taken into account
     # properly.
-    ${SALT_CALL} --local --retcode-passthrough state.sls \
+    ${SALT_CALL} --local --state-output=mixed --retcode-passthrough state.sls \
         metalk8s.node.grains \
         saltenv=metalk8s-@@VERSION \
         pillarenv=metalk8s-@@VERSION
 
-    # TODO: FIX register IP
+    # TODO: FIX registry IP
     local registry_ip
     registry_ip=$(salt-call --local grains.get metalk8s:control_plane_ip --out txt | awk '/^local\: /{ print $2 }')
-    ${SALT_CALL} --local --retcode-passthrough state.orchestrate \
-        metalk8s.orchestrate.bootstrap_without_master \
+
+    ${SALT_CALL} --local --state-output=mixed  --retcode-passthrough state.sls \
+        metalk8s.roles.bootstrap \
         saltenv=metalk8s-@@VERSION \
         pillarenv=metalk8s-@@VERSION \
-        pillar="{'registry_ip': $registry_ip}"
-
+        pillar="{'repo': {'local_mode': True}, 'registry_ip': $registry_ip}"
     set_salt_command
     sync_salt
+    # Do CA only first to push informations to the mine
+    ${SALT_CALL} --state-output=mixed --retcode-passthrough state.sls \
+        metalk8s.roles.ca \
+        saltenv=metalk8s-@@VERSION \
+        pillarenv=metalk8s-@@VERSION
+    ${SALT_CALL} --state-output=mixed --retcode-passthrough state.sls \
+        metalk8s.roles.bootstrap \
+        saltenv=metalk8s-@@VERSION \
+        pillarenv=metalk8s-@@VERSION \
+        pillar="{'registry_ip': $registry_ip}"
 
-    # shellcheck disable=SC2086
-    ${SALT_MASTER_CALL} salt-run state.orchestrate \
-        metalk8s.orchestrate.bootstrap_with_master \
+    ${SALT_MASTER_CALL} salt-run --state-output=mixed state.sls \
```

Comment:
Instead of running multiple `deployed` states, could this be bundled in one `including` others?

Reply:
same as above